ingest/expression_files/mtx.py

```diff
@@ -193,12 +193,14 @@ def get_data_start_line_number(file_handler: IO) -> int:
         """
         i = 0
         for line in file_handler:
```

Comment:
```for count, line in enumerate(file_handler):``` seems like a more elegant/pythonic way to get the line count like you want

Reply:
ad87eb7